.travis.yml

```diff
@@ -37,9 +37,12 @@ script:
 # Wait for postgres database to be running
 - 'while ! echo exit | nc localhost 5433; do sleep 10; done'
 - sleep 10
+- make APP_ENV=testing create-tables
 - make test
-deploy:
-  provider: script
-  script: 'scripts/deploy.sh staging $HOME/.ssh/id_rsa_travis'
-  on:
-    branch: master
+
+# Disable auto deployment
+#deploy:
+#  provider: script
+#  script: 'scripts/deploy.sh staging $HOME/.ssh/id_rsa_travis'
+#  on:
+#    branch: master
```

Comment:
Unclear to me why this was commented out, will check specific commits for hints.

Reply:
This was used for doing continuous delivery via travis to staging.measurements.ooni.io. Since the measurement API now requires a populated database to re-enable this we either need to have a staging snapshot of it or come up with some better way to do continous delivery to staging (if we even care to have it).

For the moment I left it commented out as a reminder that it's there and it's a thing that we may care to do in the future.